src/aux-server/aux-web/aux-player/scene/PlayerGame.ts

```diff
@@ -1139,23 +1139,77 @@ export class PlayerGame extends Game {
             sim.helper.transaction(asyncResult(e.taskId, null));
         }
     }
-
+    //MY CHANGES START HERE
     private _calculateViewportCoordinatesFromPosition(
         sim: Simulation,
         e: CalculateViewportCoordinatesFromPositionAction
     ) {
-        const portalTag = getPortalTag(e.portal);
+        let portalTag;
+        if (e.portal === 'map') {
+            portalTag = 'mapPortal';
+        } else if (e.portal === 'grid') {
+            portalTag = 'gridPortal';
+        } else if (e.portal === 'miniGrid') {
+            portalTag = 'miniGridPortal';
+        } else if (e.portal === 'miniMap') {
+            portalTag = 'miniMapPortal';
+        } else {
+            portalTag = getPortalTag(e.portal);
+        }
```

Comment:
@TroyceGowdy This change isn't needed. `getPortalTag()` does the job of getting the correct portal tag fine on its own.